src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannel.cs

```diff
@@ -18,7 +18,7 @@
 namespace System.ServiceModel.Channels
 {
     // This class is sealed because the constructor could call Abort, which is virtual
-    internal sealed class ServiceChannel : CommunicationObject, IChannel, IClientChannel, IDuplexContextChannel, IOutputChannel, IRequestChannel, IServiceChannel
+    internal /*sealed*/ class ServiceChannel : CommunicationObject, IChannel, IClientChannel, IDuplexContextChannel, IOutputChannel, IAsyncRequestChannel, IServiceChannel
```

Comment:
Great question -- I need to add a new virtual method RequestAsync, but the compiler disallows it when the class is sealed.  Given this is already internal, I think it's fine to remove the sealed.  Note the comment why it was sealed -- FxCop must have warned them.

---
In reply to: [78154483](https://github.com/dotnet/wcf/pull/1515#discussion_r78154483) [](ancestors = 78154483)